Clarifications to ways of reading and conformance

mattgarrish opened the pull request:
This pull request implements the clarifications that we discussed on the last editor's call.

For ways to read, those were:

- adding a new introduction to 3.1 Ways of reading. It gives a quick overview of the subsections and then explains that they are not treated as separate display fields. I also moved the note about always displaying the field up here.
- modifying the note for prerecorded audio to explain that it doesn't always have to be shown.
- adding a basic example of ways to read metadata for a book with text and images. Note that this will also output "Has alternative text". I could go even more basic to text only, if we don't want that output.

For conformance they were:

- explaining more clearly that detailed conformance is technical information that may not be for the average user, and detailing options for how it can be separated from the information that comes before
- adding "-details" to the IDs for all the strings that are output under detailed conformance
- changing the examples to use an open details element to show how the info can be collapsed. Maybe one could be collapsed if we want a bit of variety?

In addition, I removed the instruction to output the detailed conformance information heading from the epub and onix techniques. I replaced it with a note in the first output instruction that all the following steps are for detailed conformance and linked to the new text in the detailed conformance section for how to handle them. I also added "-details" to all the IDs in these documents, too.

I think that covers everything we discussed, but if I've forgotten anything let me know.

***

[Guidelines Preview](https://raw.githack.com/w3c/publ-a11y/update/field-defs/a11y-meta-display-guide/2.0/draft/guidelines/index.html) | [Guidelines Diff](https://services.w3.org/htmldiff?doc1=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/draft/guidelines/index.html&doc2=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://raw.githack.com/w3c/publ-a11y/update/field-defs/a11y-meta-display-guide/2.0/draft/guidelines/index.html)
[EPUB Preview](https://raw.githack.com/w3c/publ-a11y/update/field-defs/a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html) | [EPUB Diff](https://services.w3.org/htmldiff?doc1=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html&doc2=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://raw.githack.com/w3c/publ-a11y/update/field-defs/a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html)
[ONIX Preview](https://raw.githack.com/w3c/publ-a11y/update/field-defs/a11y-meta-display-guide/2.0/draft/techniques/onix-metadata/index.html) | [ONIX Diff](https://services.w3.org/htmldiff?doc1=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/draft/techniques/onix-metadata/index.html&doc2=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://raw.githack.com/w3c/publ-a11y/update/field-defs/a11y-meta-display-guide/2.0/draft/techniques/onix-metadata/index.html)

clapierre commented on a11y-meta-display-guide/2.0/draft/guidelines/index.html:
```diff
@@ -894,7 +911,40 @@ <h5>Display techniques for prerecorded audio support
+								and text)</p>
+							<p data-localization-id="ways-of-reading-nonvisual-reading-readable"
+								data-localization-mode="descriptive">All content can be read as read aloud speech or dynamic braille</p>
+						</dd>
```
are we missing the "descriptive" version for alt text?
<p data-localization-id="ways-of-reading-nonvisual-reading-alt-text" data-localization-mode="descriptive">Has alternative text descriptions for images</p>

mattgarrish replied:
Oops, yes, good catch. I cut too much from that one.

GeorgeKerscher commented:
There is a spelling error on images. It says in the first example. Publication with text and imges

GeorgeKerscher commented:
The details element is open in both examples. I was thinking the second example could be closed to demonstrate the two options. I have no strong opinion on this.

mattgarrish commented:
> The details element is open in both examples. I was thinking the second example could be closed

I actually just did the reverse. Since compact means less, I've closed it and left the descriptive example's open. I added an explanatory paragraph to highlight this.

> There is a spelling error on images.

Thanks, yes, fixed in the last update.

mattgarrish commented:
> Since compact means less, I've closed it and left the descriptive example's open.

This is supported now in the viewer app - closed details for compact display and open for descriptive.

clapierre commented:
Looks good Matt, I am wondering is there any way to have more of a visual spacing between each Compact and Descriptive example, they seem to flow into each other. This would affect every Example here.

mattgarrish commented:
> is there any way to have more of a visual spacing between each Compact and Descriptive example, they seem to flow into each other

Ya, agree. I've added some CSS to increase the margin above. Might take a few minutes before it shows up in the previews, though.

clapierre reviewed (approved):
Much better thanks Matt

gregoriopellegrino commented:
If I have seen it correctly, it seems to me that it is indicated that "Ways of reading" should always be shown, but we know that for "Prerecorded audio" information this is not the case.
